src/plot_api/plot_template.js

```diff
@@ -203,6 +203,8 @@ exports.arrayTemplater = function(container, name, inclusionAttr) {
         // it's explicitly marked visible - in which case it gets NO template,
         // not even the default.
         out[inclusionAttr] = itemIn[inclusionAttr] || false;
+        // special falsy value we can look for in validateTemplate
+        out._template = false;
```

Comment:
This case is just special because we were told to look for a template by name and it wasn't found.

Otherwise `_template` can be `null` (no template was found but that's fine) or `undefined` (we don't need to propagate the template into this container).